openmmtools/multistate/multistatesampler.py

```diff
@@ -1774,6 +1772,15 @@ def _display_cuda_devices():
         cuda_devices_list = [entry.split(',') for entry in cuda_query_output.split('\n')]
         logger.debug(f"CUDA devices available: {*cuda_devices_list,}")
 
+    def _flatten_moves_iterator(self):
```

Comment:
Add a docstring (I know it is a private method but I it would be nice) describing what this method does.